Makefile

```diff
@@ -112,6 +112,15 @@ rgbfix: ${rgbfix_obj}
 rgbgfx: ${rgbgfx_obj}
 	$Q${CC} ${REALLDFLAGS} ${PNGLDFLAGS} -o $@ ${rgbgfx_obj} ${PNGLDLIBS}
 
+# Dependencies
+rgbasm_deps	:= ${rgbasm_obj:.o=.d}
```

Comment:
I thought substitutions like this weren't supported by BSD Make?

Reply:
Oh, it didn't occur to me that it could be the case. Honestly, no idea.